src/demo/DemoTestClientService.java

```diff
@@ -795,4 +802,48 @@ public static void testUpdateGroup(GroupExpanded groupExpanded) throws MambuApiE
 		System.out.println("Group Updated. Name=" + groupExpanded.getGroup().getGroupNameWithId() + "\tName and Id="
 				+ updatedGroupExpaneded.getGroup().getGroupNameWithId());
 	}
+
+	// Test posting comments and getting comments for all supported entities
+	public static void testPostGetComments() throws MambuApiException {

```

Comment:
Please extract to separate class since this is a test for all comments and not only client comments.